tests/unit/cartography/test_util.py

```diff
@@ -13,3 +17,69 @@ def test_run_analysis_job_custom_package(mocker):
     read_text_mock = mocker.patch('cartography.util.read_text')
     util.run_analysis_job('test.json', mocker.Mock(), mocker.Mock(), package='a.b.c')
     read_text_mock.assert_called_once_with('a.b.c', 'test.json')
+
+
+def test_aws_handle_regions(mocker):
+    # no exception
+    @aws_handle_regions
+    def f(a, b):
```

Comment:
I know this is a test func but I think it'd be more readable if we didn't use single letter variable names. How about these names?

'f()' --> 'happy_path()'

'g()' --> 'raises_client_error()'

'h()' --> 'custom_return_value()'

'i()' --> 'unhandled_exc()'

'j()' --> 'other_excs()'

Reply:
Changed.